django_prometheus/middleware.py

```diff
@@ -1,5 +1,6 @@
 from prometheus_client import Counter, Histogram
 from django_prometheus.utils import Time, TimeSince, PowersOf
+from django.utils.deprecation import MiddlewareMixin
```

Comment:
Maybe you could try something like this?

    import django
    if django.VERSION >= (1, 10, 0):
        from django.utils.deprecation import MiddlewareMixin
    else:
        MiddlewareMixin = object